compiler/driver.ml

```diff
@@ -989,6 +1007,80 @@ module Commands = struct
         $ Cli.Flags.optimize
         $ Cli.Flags.check_invariants)
 
+  let depends options includes prefix extension extra_files =
```

Comment:
Can you remove the `Makefile` command of the Catala compiler now that we have this?

Reply:
Yes; it's not completely equivalent yet (no Makefile-style output, but support for modules) but the two should certainly be merged.
Ninja actually supports dependency specs in the Makefile style, so we could leverage that at some point (at the moment, a given file is likely to get parsed many times just to extract its dependencies...)